### ISSUE TITLE
suggest not use 0.11.1 with a big bug in multisig

### ISSUE THREAD

**Issue by sampleblood** [ethereum/mist#4103]
please fix it !!!!

**Comment by ChristBKK** [comment]
yes already mentioned. 0.10 works , 0.11 not!   I am waiting for a release for over a month now :)  I really hope that the Ethereum foundation gives more priority to MIST
